project/Build.scala

```diff
@@ -99,7 +100,11 @@ object ElasticMQBuild extends Build {
   lazy val core: Project = Project(
     "elasticmq-core",
     file("core"),
-    settings = buildSettings ++ Seq(libraryDependencies ++= Seq(jodaTime, jodaConvert, akka2Actor, akka2Testkit) ++ common)
+    settings = buildSettings ++ Seq(
+      libraryDependencies ++= Seq(jodaTime, jodaConvert, akka2Actor, akka2Testkit) ++ common,
+      coverageMinimum := 94,
```

Comment:
We can monitor code coverage but I would be a bit wary of introducing required minimums. It would be good to know that coverage decreased, but ultimately I would trust the developers that they know what they are doing :)

Reply:
FWIW, this will only affect the `coverageReport` task, after it generated the reports, it will exit with a RunTimeException. As that's usually the last task in the chain, it shouldn't affect much. Tasks such as `sbt test` will still work as before. That said, I can see why it can be a bit scary, so I'll take it out. We can always add it back in later when we're a bit more comfortable with it.

Reply:
Thanks for the explanation :) By the way - maybe you could add the information on how to generate the coverage reports (as in the PR description) to the readme?

Reply:
Good idea! Added that in :)